ppr-ui/src/composables/useInputRules.ts

```diff
@@ -55,13 +55,23 @@ export const useInputRules = () => {
   }
 
   const isEmail = (): Array<Function> => [
-    (v: string) => !!v || 'Email address is required',
+    (v: string) => !!v || 'Enter an email address',
     (v: string) => {
       const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-      return pattern.test(v) || 'Valid email is required'
+      return pattern.test(v) || 'Enter a valid email address'
     }
   ]
 
+  const isPhone = (
+    maxDigits: number = null,
+    customMsg: string = null
+  ): Array<Function> => {
+    return [
+      v => ((v && maxDigits) ? v.length >= maxDigits : true) || 'Minimum 10 characters',
+      v => (v ? /^\d+$/g.test(v.replace('(', '').replace(') ', '').replace('-', '')) : true) || `${customMsg || 'Must contain numbers only'}`
+    ]
+  }
```

Comment:
Arn't phone numbers strings to our specs that can also contain characters like `-` `()` and whatnot?

Reply:
Very good question... Phone numbers can be only numbers.  The mask is only for user validation and display on review form.  Doug made a comment regarding the phone field in db2 is only 10 digits... I am going to remove the extra characters before writing to store...

Reply:
https://bcregistry-demo.apigee.io/docs/mhrproxy/1/types/owner

![Screen Shot 2022-08-24 at 3 03 27 PM](https://user-images.githubusercontent.com/53542131/186531846-6178f34a-28ac-4c5f-afa7-238866c2af16.png)


The spec states that phone numbers are strings so i'm confused.
Was there a request for the spec to change as well?

Where does the requirement for number only come from?

Reply:
🤔 
Sorry, it's a bit misleading, the spec is saying it is a string but formatted only containing digits so i suppose that is where the confusion might be coming from. 
Can you confirm that the end result of this fields INPUT is as the example is above? 
If so, than all good 👍

Reply:
The end result is a string..